api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -871,9 +872,21 @@ public ResponseEntity<RecentWorkspaceResponse> updateRecentWorkspaces(
 
   @Override
   public ResponseEntity<WorkspaceResourceResponse> getWorkspaceResources(
-      String workspaceNamespace,
-      String workspaceId,
-      WorkspaceResourcesRequest workspaceResourcesRequest) {
+      String ns, String id, WorkspaceResourcesRequest wrr) {
+    return getWorkspaceResourcesImpl(ns, id, wrr.getTypesToFetch());
```

Comment:
The shortened variable names help to make this line stand out, which is the only bit of real work done in this method.